<launch>

<!--    -       -->
<!--  arg       --> 
<!--    -       -->



<!--    -               -->  
<!--  rosparam / param  --> 
<!--    -               -->



<!--    -       -->
<!--  node      --> 
<!--    -       -->  



  <!-- Start the test -->
  <node name="pomdp_stop" pkg="itia_kinova" type="pomdp_stop" output="screen" >
  </node>

  
<!--    -       -->
<!--  include   --> 
<!--    -       -->
  
  
</launch>
